geo/overturemaps/download/wrapper.py

```
@@ -0,0 +1,25 @@
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+extra = snakemake.params.get("extra", "")
+bbox = snakemake.params.get("bbox", "")
+if bbox:
+    if isinstance(bbox, list):
+        bbox = ",".join(map(str, bbox))
+    bbox = f"--bbox {bbox}"
+
+shell(
+    "overturemaps download"
+    f" -f {snakemake.params.format}"
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)